perl/javadoc/io/perl/api/Perl.html

```
@@ -157,7 +164,26 @@ <h3>stop</h3>
+</div>
+</section>
+</li>
+<li>
+<section class="detail" id="stop(io.perl.api.BenchmarkTermination)">
+<h3>stop</h3>
+<div class="horizontal-scroll">
+<div class="member-signature"><span class="modifiers">default</span>&nbsp;<span class="return-type">void</span>&nbsp;<span class="element-name">stop</span><wbr><span class="parameters">(<a href="BenchmarkTermination.html" title="enum class in io.perl.api">BenchmarkTermination</a>&nbsp;termination)</span></div>
+<div class="block">Stops the benchmark while preserving an orchestrator-provided successful
+completion reason in the final lifecycle log.
+
+<p>The default implementation retains compatibility with alternate PerL
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 181.

(tag-pair)